metasploit-framework.gemspec

```diff
@@ -136,7 +136,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'net-ssh'
   spec.add_runtime_dependency 'ed25519' # Adds ed25519 keys for net-ssh
   spec.add_runtime_dependency 'bcrypt_pbkdf'
-  spec.add_runtime_dependency 'ruby_smb', '~> 1.1'
+  spec.add_runtime_dependency 'ruby_smb', '~> 2.0'
```

Comment:
Does `Gemfile.lock` need updated too? 🤔

Reply:
Yes, absolutely. When this [PR](https://github.com/rapid7/ruby_smb/pull/156) gets merged, we will release a new version of `ruby_smb` and I will update the `gemspec` file.

Reply:
Done! `ruby_smb` version [`2.0.1`](https://rubygems.org/gems/ruby_smb/versions/2.0.1) has been released and incorporates the referenced changes.